tasks/ceph.py

```diff
@@ -246,8 +246,12 @@ def valgrind_post(ctx, config):
                 log.error('saw valgrind issue %s in %s', kind, file)
                 valgrind_exception = Exception('saw valgrind issues')
 
-        if valgrind_exception is not None:
-            raise valgrind_exception
+        if config.get('expect-valgrind-errors'):
```

Comment:
Is ``expect-valgrind-errors`` a new setting? If so, I'd suggest using underscores if you don't mind.